docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -40,7 +40,37 @@ language into our day-to-day work.
 
 1.0 Overview 
 ++++++++++++
-Add an introductory paragraph here (include visual of 'What this is vs. what this isn't').
+As part of our team and organization's larger DEI efforts, we are committed to researching
+and building a style guide so as to incorporate inclusive language into our day-to-day work
+and to address the harmful conflation of sex and gender in public health research and data. 
+We hope this page will serve as a useful guide to members of the Simulation Science team as
+well as other teams at IHME!
+
+On this page you will find our guiding principles, a glossary of commonly used terms related
+to BLISS, recommendations for how to address sex/gender-related limitations in external data
+sources, discussion of the implications of imperfect proxy measures with regards to sex/gender,
+examples of how our team has incorporated BLISS into our work process, and more! 
+
+If you have any questions or comments about the content on this page - please reach out! You can
+contact us directly `on GitHub <https://github.com/ihmeuw/vivarium_research/issues?q=is%3Aopen+is%3Aissue+label%3Abliss>`_ or anonymously 
+on our `team HUB page <https://hub.ihme.washington.edu/display/COS/BLISS%3A+Building+Language+Inclusivity+into+Simulation+Science>`_.  
```

Comment:
I guess this only works for people who have HUB access, such as IHME staff

Reply:
Yes, we link to internal-only pages pretty frequently in Vivarium Research, and usually don't have any kind of special marker on these links. Seems like something that would be good to have more of a policy around, but not unique to this page.